app/assets/stylesheets/components/_organizations.scss

```diff
@@ -1,30 +1,99 @@
-.organization-details {
-  .section {
-    border-top: $base-border;
-    margin-top: $base-spacing;
-    padding-top: $base-spacing / 2;
+$actions-height: 3 * $base-spacing;
 
-    &.location {
-      border: $base-border;
-      padding: $base-spacing / 2;
+.organization {
+  &-details {
+    .section {
+      border-top: $base-border;
+      margin-top: $base-spacing;
+      padding-top: $base-spacing / 2;
+
+      &.location {
+        border: $base-border;
+        padding: $base-spacing / 2;
+      }
+    }
+
+    .subsection {
+      margin-top: $base-spacing;
+    }
+
+    .column {
+      @include span-columns(4.5 of 9);
+
+      &:last-child {
+        @include omega;
+      }
+    }
+
+    .map {
+      @include fill-parent;
+      height: 20em;
+      margin-bottom: $base-spacing;
     }
   }
 
-  .subsection {
-    margin-top: $base-spacing;
+  &-modal {
+    @include padding($base-spacing);
+    background-color: #fff;
+    box-shadow: 0 0 6px rgba(#000, 0.3);
+    left: 0;
+    margin: $base-spacing*2 $base-spacing*4;
+    position: absolute;
+    right: 0;
   }
 
-  .column {
-    @include span-columns(4.5 of 9);
+  &-listing {
+    $listing-height: 2 * $base-spacing;
+    padding: $base-spacing/2 0;
+    position: relative;
+
+    &:not(:last-child) {
+      border-bottom: $base-border;
+    }
 
     &:last-child {
-      @include omega;
+      margin-bottom: $base-spacing;
+    }
+  }
+
+  &-invalid {
+    &:before {
+      $size: 24px;
+      background: image_url("warning.svg") center no-repeat;
```

Comment:
Name of function `image_url` should be written in all lowercase letters with hyphens instead of underscores